easybuild/easyconfigs/s/ScaLAPACK/ScaLAPACK-2.0.2-gompic-2016.08-OpenBLAS-0.2.15-LAPACK-3.6.0.eb

```diff
@@ -0,0 +1,25 @@
+name = 'ScaLAPACK'
+version = '2.0.2'
+
+homepage = 'http://www.netlib.org/scalapack/'
+description = """The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
+ redesigned for distributed memory MIMD parallel computers."""
+
+toolchain = {'name': 'gompic', 'version': '2016.08'}
+toolchainopts = {'pic': True}
+
+source_urls = [homepage]
+sources = [SOURCELOWER_TGZ]
+
+blaslib = 'OpenBLAS'
+blasver = '0.2.15'
+blassuff = '-LAPACK-3.6.0'
+
+versionsuffix = "-%s-%s%s" % (blaslib, blasver, blassuff)
+
+dependencies = [(blaslib, blasver, blassuff, ('GCC', '4.9.3-2.25'))]
```

Comment:
I was trying really hard to match existing conventions here. See, for example, this comparison with the existing ScaLAPACK easybuild for the 2016a toolchain:

```
--- ScaLAPACK-2.0.2-gompi-2016a-OpenBLAS-0.2.15-LAPACK-3.6.0.eb 2016-08-21 12:51:24.231715518 -0400
+++ ScaLAPACK-2.0.2-gompic-2016.08-OpenBLAS-0.2.15-LAPACK-3.6.0.eb      2016-08-24 23:49:14.932432277 -0400
@@ -5,7 +5,7 @@
 description = """The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
  redesigned for distributed memory MIMD parallel computers."""

-toolchain = {'name': 'gompi', 'version': '2016a'}
+toolchain = {'name': 'gompic', 'version': '2016.08'}
 toolchainopts = {'pic': True}

 source_urls = [homepage]
```